.github/workflows/build_wheels.yml

```diff
@@ -15,38 +15,52 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macOS-10.15, ubuntu-20.04, windows-2019]
-
+        os: [ubuntu-latest, macos-latest, windows-latest]
     steps:
       - uses: actions/checkout@v2
 
-      # Used to host cibuildwheel
       - uses: actions/setup-python@v2
 
       - name: Install cibuildwheel
         run: python -m pip install cibuildwheel==1.9.0
 
+      - name: Clone pybind11 repo
+        run: git clone https://github.com/pybind/pybind11.git
```

Comment:
unlikely, but possible that getting the latest pybind might cause problems.  Perhaps clone a tagged release instead?

Reply:
Very good point, will make that change